terraform/environments/digital-prison-reporting/modules/dms_s3/config/nomis-replication-settings.json

```diff
@@ -120,7 +120,7 @@
   "FullLoadSettings": {
     "CommitRate": 10000,
     "StopTaskCachedChangesApplied": false,
-    "StopTaskCachedChangesNotApplied": false,
+    "StopTaskCachedChangesNotApplied": true,
```

Comment:
This stops the DMS task after the load is compete so the batch job can list and process all the load files. This will avoid data loss.